### PR TITLE
feat(changelog): private-networks-changed-virtual-private-cloud-is 2023-07-18

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Benedikt Rollik

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+status: changed
+author:
+    fullname: 'Join the #virtual-private-cloud channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-18
+category: network
+product: private-networks
+---
+
+Formerly named Private Networks, VPC allows you to secure quickly and seamlessly all the resources of your application's architecture from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the AZs by default.
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:12]:
```suggestion
VPC allows you to quickly and seamlessly secure all your application architecture's resources of from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.
```

File: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- By default, each region will have at least one VPC, and you can create more if needed
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:13]:
```suggestion
- By default, each region will have at least one VPC. You can create more if needed.
```

File: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+status: changed
+author:
+    fullname: 'Join the #virtual-private-cloud channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-18
+category: network
+product: private-networks
+---
+
+VPC allows you to quickly and seamlessly secure all your application architecture's resources of from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.
```
Comment (SamyOubouaziz) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:12]: "resources of from" -> "resources from"

Comment (nerda-codes) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:12]:
```suggestion
VPC allows you to quickly and seamlessly secure all your application architecture's resources from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- By default, each region will have at least one VPC. You can create more if needed.
+- Each VPC can have multiple Private Networks, each with its own distinct CIDR blocks
```
Comment (SamyOubouaziz) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:14]: missing full stop at the end of the sentence ?

File: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+- VPC will now include a DHCP server for each Private Network
```
Comment (SamyOubouaziz) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:15]: Inconsistent end punctuation

File: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+author:
+    fullname: 'Join the #virtual-private-cloud channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-18
+category: network
+product: private-networks
+---
+
+VPC allows you to quickly and seamlessly secure all your application architecture's resources from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.
+- By default, each region will have at least one VPC. You can create more if needed.
```
Comment (bene2k1) [inline changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx:13]:
```suggestion
- By default, each region will have at least one VPC. You can create more if needed
```

**Pull request merged**